lib/magma.gi

```diff
@@ -184,16 +184,17 @@ InstallImmediateMethod( IsCommutative,
     fi;
     end );
 
-InstallImmediateMethod( IsCommutative,
-    IsMagmaWithInverses and IsAssociative
-                        and HasGeneratorsOfMagmaWithInverses, 0,
-    function( M )
-    if Length( GeneratorsOfMagmaWithInverses( M ) ) = 1 then
-      return true;
-    else
-      TryNextMethod();
-    fi;
-    end );
+# this is now set when creating groups
```

Comment:
This should say "when creating magmas". But even then it's not quite true: If a magma is created in a non-standard way, it may not be set. So, like with most of the other immediate methods you remove/commented out, I'd prefer to instead turn it into a regular method.